modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -520,5 +534,10 @@ describe('TSS Ecdsa Utils:', async function () {
     return params.keyChain;
   }
 
+  async function nockGetTxRequest(params: {walletId: string, txRequestId: string, response: unknown }): Promise<nock.Scope> {
```

Comment:
we already have a nock function defined in `modules/bitgo/test/v2/unit/internal/tssUtils/common.ts `?

Reply:
the persist is causing issues even when disable it. I'll use the other nock which part of helpers